Client/Frontend/ClientPreferences.swift

```diff
@@ -210,7 +210,7 @@ extension Preferences {
     /// The last played item time
     static let lastPlayedItemTime = Option<Double>(key: "playlist.last.played.item.time", default: 0.0)
     /// Whether to play the video when controller loaded
-    static let firstLoadAutoPlay = Option<Bool>(key: "playlist.firstLoadAutoPlay", default: false)
+    static let firstLoadAutoPlay = Option<Bool>(key: "playlist.firstLoadAutoPlay2", default: true)
```

Comment:
Why are we introducing a new preference for this instead of just changing the default? Someone who's previously turned it on then back off for instance will have it reset to on again

Reply:
You're right. It will only apply to new users though. I wasn't sure whether to force it for all users and have them turn it off if they want. But yeah, the alternative is better where it applies to only new users. I'll fix it :)

Reply:
This should be the place https://github.com/brave/brave-ios/blob/development/Client/Migration/Migration.swift#L52